bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/TableColumnsAdjuster.scala

```diff
@@ -0,0 +1,84 @@
+package edu.gemini.catalog.ui
+
+import javax.swing.table.TableColumn
+import scala.swing.Table
+
+import scalaz._
+import Scalaz._
+
+/**
+ * Support calculating column widths and can adjust them to the outer width
+ */
+trait TableColumnsAdjuster { this: Table =>
```

Comment:
This is very similar to `resizeTableColumns` in `SequenceTabUtil`. Maybe we can bring those together at some point (not saying as part of this PR ;).